README.md

```diff
@@ -7,6 +7,82 @@ klifs_utils
 
 Utility functions to work with KLIFS data
 
+
+### KLIFS
+
+The `klifs_utils` package offers functions to interact with data from KLIFS (remotely and locally).
+
+What is KLIFS and who created it?
+
+> Kinase-Ligand Interaction Fingerprints and Structures database (KLIFS), developed at the Division of Medicinal Chemistry - VU University Amsterdam, is a database that revolves around the protein structure of catalytic kinase domains and the way kinase inhibitors can interact with them
+* KLIFS database: https://klifs.vu-compmedchem.nl
+* KLIFS online serive: https://klifs.vu-compmedchem.nl/swagger
+* KLIFS citation
+  * Description of the database itself [(*J. Med. Chem.* (2014), **57**, 2, 249-277)]()
+  * Description of the online service [(*Nucleic Acids Res.* (2016), **44**, 6, D365–D371)]()
+
+
+
+### Installation
+
+1. Download GitHub repo:
+
+    ```bash
+    git clone https://github.com/dominiquesydow/klifs_utils.git
+    ```
+    
+    The library lives now in this example path `/path/to/klifs_utils`.
+    
+2. Install and activate conda environment `klifs`:
+    ```bash
+    conda env create -f /path/to/klifs_utils/devtools/env.yaml
```

Comment:
That would be really cool - I think it is not supported yet but it is an open issue: https://github.com/conda/conda/issues/5562 

However, since - at the moment - the user needs to download the `klifs_utils` repo anyways to install it, I think it is ok to create the conda environment from file. Still, would be cool to allow it via URL, too.

Of course you are right about the README file path to `env.yaml` :) It lives correctly in `conda_envs` but I put the wrong path here. Thanks!